blocks/api/post.pegjs

```diff
@@ -161,19 +161,19 @@ function partition( predicate, list ) {
 //////////////////////////////////////////////////////
 
 Block_List
-  = pre:$(!Token .)*
-    ts:(t:Token html:$((!Token .)*) { /** <?php return array( $t, $html ); ?> **/ return [ t, html ] })*
+  = pre:$(!Block .)*
+    bs:(b:Block html:$((!Block .)*) { /** <?php return array( $b, $html ); ?> **/ return [ b, html ] })*
     post:$(.*)
-  { /** <?php return peg_join_blocks( $pre, $ts, $post ); ?> **/
-    return joinBlocks( pre, ts, post );
+  { /** <?php return peg_join_blocks( $pre, $bs, $post ); ?> **/
+    return joinBlocks( pre, bs, post );
   }
 
-Token
-  = Block_Void
+Block
+  = Block_Empty
```

Comment:
It’s worth considering the context in which void was chosen as a name. It borrows from the HTML spec which specifies certain tags as void and those are not allowed to have a closing tag - hr, img, br, etc... in Gutenberg it’s optional the distinction between empty and void but the intention was to convey void elements, not to present an optimization for empty elements